server/index.js

```diff
@@ -6,7 +6,7 @@ app.use(express.static('public'));
 app.use(express.json());
 
 app.all('*', (req, res) => {
-
+  console.log('body: ', req);
```

Comment:
I'm wondering if we need to remove this console.log eventually. Again I think it's totally cool to have it here for now!

Reply:
I forgot I had these in there. Let me push another commit so that these don't bother anyone later